spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/DefaultSkipperClient.java

```diff
@@ -135,6 +137,40 @@ public Release rollback(String releaseName, int releaseVersion) {
 	}
 
 	@Override
+	public List<Release> getReleases(String releaseNameLike) {
+		ParameterizedTypeReference<List<Release>> typeReference = new ParameterizedTypeReference<List<Release>>() {
+		};
+		String url;
+		if (StringUtils.hasText(releaseNameLike)) {
+			url = String.format("%s/%s/%s/%s", baseUrl, "releases", "listByName", releaseNameLike);
+		}
+		else {
+			url = String.format("%s/%s/%s", baseUrl, "releases", "list");
+		}
+		return this.restTemplate.exchange(url, HttpMethod.GET, null, typeReference, new HashMap<>()).getBody();
+	}
+
+	@Override
+	public List<Release> getRevisions(String releaseName, String maxRevisions) {
+		ParameterizedTypeReference<List<Release>> typeReference = new ParameterizedTypeReference<List<Release>>() {
+		};
+		Map<String, Object> parameters = new HashMap<>();
+		String url = String.format("%s/%s/%s/%s/%s", baseUrl, "releases", "revisions", releaseName, maxRevisions);
+		return this.restTemplate.exchange(url, HttpMethod.GET, null, typeReference, parameters).getBody();
+	}
+
+	@Override
+	public Resources<Release> getRevisions(String releaseName) {
```

Comment:
Why does this return Resource<Release> as the only user of it, in the command, asks for getContent().